applications/ecallmgr/src/ecallmgr_fs_node_command.erl

```diff
@@ -61,22 +65,28 @@ reply_success(JObj, Response) ->
     kz_amqp_worker:cast(API, fun(P) -> kapi_switch:publish_reply(Queue, P) end).
 
 
--spec send_http(atom(), file:filename(), ne_binary(), ne_binary(), kz_json:object()) -> 'ok'.
-send_http(Node, File, Url, Method, JObj) ->
+-spec send_http(atom(), file:filename(), ne_binary(), ne_binary(), kz_json:object(), boolean()) -> 'ok'.
```

Comment:
warn_contract_types           ecallmgr_fs_node_command.erl:68: Invalid type specification for function ecallmgr_fs_node_command:send_http/6. The success typing is (_,binary(),binary(),<<_:8,_:_*1>>,_,_) -> any()